support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -1220,3 +1203,30 @@ form {
 .editorialise-amounts-test-copy:empty {
   display: none;
 }
+
+// Vertical contributions tabs test
+.form__radio-group--mobile-vertical-tabs .form__radio-group-list {
+  flex-flow: column-reverse nowrap;
```

Comment:
with sass you can put all of this inside the parent

```sass
.form__radio-group--mobile-vertical-tabs {
   .form__radio-group-list {

  }

  .form__radio-group-list--border {

  }
}
```

Reply:
@joelochlann that's how I would normally have written it but I thought the Guardian doesn't like nested styles-- there's no other nested styles in the sheet (I think) except for media queries

Reply:
@joelochlann nevermind - i will refactor